roles/tftp-server/templates/preseed.cfg.j2

```diff
@@ -331,7 +331,7 @@ d-i apt-setup/local0/key string file:///hd-media/ac583520.asc
 tasksel tasksel/first multiselect standard
 
 # Individual additional packages to install
-d-i pkgsel/include string openssh-server curl
+d-i pkgsel/include string openssh-server curl python python3
```

Comment:
for ansible.  not sure if it wanted v2 or 3.

moved from late_command.sh apt install python 

I think it is a safe assumption that we:
 want the box ready for ansible. (all cases)
 may not use late_command.sh (some cases)